packages/hdwallet-core/src/ethereum.ts

```diff
@@ -126,6 +140,12 @@ export interface ETHWalletInfo extends HDWalletInfo {
    */
   ethSwitchChain?(chain_id: number): Promise<void>;
 
+  /**
+   * Add an Ethereum chain to user's wallet
+   * https://eips.ethereum.org/EIPS/eip-3085
+   * */
+  ethAddChain?(params: AddEthereumChainParameter): Promise<void>;
```

Comment:
Should this be `evmAddChain` or `ethBasedAddChain` instead?

Reply:
> Should this be `evmAddChain` or `ethBasedAddChain` instead?

I agree with you in principle but it would break the established pattern of all the functions being named `ethXXX`. For now, I don't think it's worth introducing an inconsistency.